dotnet/src/AutoGen.SemanticKernel/SemanticKernelAgent.cs

```diff
@@ -29,39 +29,41 @@ namespace AutoGen.SemanticKernel;
 /// </summary>
 public class SemanticKernelAgent : IStreamingAgent
 {
-    private readonly Kernel _kernel;
+    public string Name { get; }
+
+    private readonly ChatCompletionAgent _chatCompletionAgent;
     private readonly string _systemMessage;
-    private readonly PromptExecutionSettings? _settings;
 
-    public SemanticKernelAgent(
-        Kernel kernel,
+    public SemanticKernelAgent(ChatCompletionAgent chatCompletionAgent, string name,
+        string systemMessage = "You are a helpful AI assistant")
```

Comment:
Can we use the `chatCompletionAgent.Name` to set the `this.Name` property instead of passing another `name` in the constructor